src/test/java/nextstep/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -23,6 +54,41 @@ void createLine() {
     @DisplayName("지하철 노선 목록 조회")
     @Test
     void getLines() {
+        // given
+        Map<String, String> params = new HashMap<>();
+        params.put("color", "bg-red-600");
+        params.put("name", "신분당선");
+
+        RestAssured.given().log().all()
+            .body(params)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when()
+            .post("/lines")
+            .then().log().all()
+            ;
```

Comment:
분리가 충분히 이뤄졌다면, 이 given API도 이렇게 줄어들 수 있을꺼에요.
```suggestion
        LineSteps.지하철_노선_생성_요청(신분당선); //LineSteps는 static import로 생략 가능
```